iroh-net/src/derp/map.rs

```diff
@@ -26,6 +27,37 @@ impl DerpMap {
         ids
     }
 
+    /// Returns an `Iteratore` over all known regions.
```

Comment:
```suggestion
    /// Returns an `Iterator` over all known regions.
```
it felt very Italian :ok_hand:

Reply:
🤣